Subject: virt-manager: list virtual networks when creating new QEMU Session VM
From: Pavel Hrdina phrdina@redhat.com Mon Mar 10 19:26:44 2025 +0100
Date: Tue Mar 18 20:33:33 2025 +0100:
Git: 714d5f1afdd58d5e908d420c4fbd8183edf47aa7

Using qemu-bridge-helper QEMU Session VMs are now able to use host
bridge interfaces. Currently only interface named virbr0 is allowed by
default but it is possible to change it in `/etc/qemu/bridge.conf`.

We will still keep the usermode network as default.

Resolves: https://github.com/virt-manager/virt-manager/issues/863
Signed-off-by: Pavel Hrdina <phrdina@redhat.com>

--- a/virtManager/device/netlist.py
+++ b/virtManager/device/netlist.py
@@ -149,7 +149,6 @@ class vmmNetworkList(vmmGObjectUI):
         add_usermode = False
         if self.conn.is_qemu_unprivileged():
             log.debug("Using unprivileged qemu, adding usermode net")
-            vnets = []
             default_bridge = None
             add_usermode = True
 
